### DISCUSSION TITLE
Prom datasource: Explore: Query error indicates which query is bad

### DISCUSSION THREAD

**Discussion by samjewell** [grafana/grafana#46155] [Data Sources]
for the Prometheus datasource explore interface:

I wrote 2 queries, one of which has a syntax error. But I can't see which query I need to fix from the UI: Neither the error message text, nor the layout of the page tells me which query needs to be fixed.

Could we possibly move the error up the page, closer to the bad query? See screenshot

![image (4)](https://user-images.githubusercontent.com/2903904/156557263-96a1f879-f35d-4d09-9d95-3c35901c60ea.png)
.

**Comment by ifrost** [comment]
This should be addressed by https://github.com/grafana/grafana/pull/47613

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by samjewell** [comment]
This actually has been fixed.
